test/gtest/uct/test_pending.cc

```diff
@@ -426,6 +425,10 @@ UCS_TEST_P(test_uct_pending, pending_ucs_ok_dc_arbiter_bug)
     /* idx 0 is setup in initialize(). only need to alloc request */
     for (i = 1; i < N; i++) {
         m_e1->connect(i, *m_e2, i);
+        if (!(i % max_listen_connections())) {
```

Comment:
minor: can be replaced with another loop to avoid `%` every iteration:
```
for (i = 1; i < N; i++) {
	for (j = 0; j < max_listen_connections(); j++) {
		m_e1->connect(i, *m_e2, i);
	}
	flush();
}
```

Reply:
done